tests/compas/geometry/test_point.py

```diff
@@ -1,8 +1,20 @@
+import compas
 from compas.geometry import Point
 
 
+def test_data():
+    if compas.IPY:
+        return
```

Comment:
it would be much much better if we don't skip ipy here... 😇

Reply:
i know but `schema` is not available for ironpython...